lips/lip-2.md

```diff
@@ -0,0 +1,119 @@
+---
+lip: 2
+title: Libra Roles and Permissions
+author: Sam Blackshear, Tim Zakian
+status: Draft
+type: Informational
+created: 06/26/2020
+---
+
+# Summary
+---
+This LIP describes the conceptual model and implementation of access control on the Libra blockchain.
+
+---
+# Abstract / Motivation
+---
+
+Libra uses a variant of [role-based access control](https://en.wikipedia.org/wiki/Role-based_access_control) (RBAC) to restrict access to sensitive on-chain operations.
+
+A *role* is an entity with some authority in the Libra system. Every account in the Libra system is created with a single, immutable role that is granted at the time the account is created. Creating an account with a particular role is a privileged operation (e.g., only an account with the ParentVASP role can create an account the ChildVASP role). In some cases, the role is globally unique (e.g., there is only one account with the LibraRoot role). In other cases, there may be many accounts with the given role (e.g., ChildVASP).
```

Comment:
Nit: (e.g., only an account with the ParentVASP role can create an account _with_ the ChildVASP role)